packages/performance/README.md

```diff
@@ -94,3 +97,7 @@ The `Navigation` object represents a full navigation, either from a full-page re
   - `metadata`: an object with arbitrary key-value pairs providing additional context
 
 `Navigation` also provides a number of utility methods for gathering more actionable information, such as `eventsByType` for filtering events to a particular type, or `totalDownloadSize` for the cumulative size of all requested resources. Please consult the TypeScript definitions for a full listing of these methods.
+
+### Events
+
+Read about the list of events `Performance` and `Navigation` class supports and what they do [here](docs/events).
```

Comment:
I don't think this section has enough explanation for why you would want to read about events. Maybe something like:

> The `Performance and `Navigation` classes both deal with `Event` objects with which contain information about the timing of specific milestones during the course of a user's browser session. 

I'd also sort of rather just nest the list of events in the same file since it feels weird to give them their own file but nothing else.